Sources/Kevin/Core/Extensions/UIImage+Networking.swift

```diff
@@ -20,6 +20,7 @@ extension UIImageView {
         
         if let cachedImage = imageCache.object(forKey: urlString as NSString)  {
             self.image = cachedImage
+            self.alpha = 1.0
```

Comment:
who sets alpha 0 in the first place?

Reply:
Hmm, alpha is set just before making a network request to load image and I wonder what is the point of it.

Also, I wonder if we are protected somehow with loading images when tableCells get reused. Maybe we have a possible bug here but just didn't notice it because of small amount of items in table.

Reply:
Missed that fact down the code that it is setting alpha to 0